transifex/native/django/management/common.py

```diff
@@ -47,6 +52,8 @@ def add(self, source_string):
         key = generate_key(source_string.string, source_string.context)
         if key not in self.strings:
             self.strings[key] = source_string
+        else:
+            self.strings[key].occurrences = source_string.occurrences
```

Comment:
Minor:  This will also probably work:

```python
self.strings.setdefault(key, source_string).occurrences = source_string.occurrences
```

Reply:
@kbairak that is an updated on existing code. I have to replace all lines from 53 to 56 and the code is not going to be self explanatory.